package/Dockerfile.dapper-base

```
@@ -39,7 +39,7 @@ RUN dnf -y distrosync --nodocs --setopt=install_weak_deps=False && \
                    gcc git-core curl moby-engine make golang kubernetes-client \
                    findutils upx jq golang-x-tools-goimports ShellCheck shflags && \
     dnf -y remove libsemanage && \
-    rpm -e --nodeps selinux-policy-targeted && \
+    rpm -qa "selinux*" | xargs rpm -e --nodeps && \
```

```suggestion
    rpm -qa "selinux*" | xargs rpm -r -e --nodeps && \
```

so this doesn’t fail if `rpm -qa` doesn’t output anything ;-).

---

Are you sure it's `-r`?

It seems to fail with
```
rpm: arguments to --root (-r) must begin with a /
```

---

Oops, sorry, wrong place, it’s meant for `xargs`: `xargs -r rpm -e --nodeps && \`